LocalStoreSimulation/src/ro/scit/menu/Menu.java

```diff
@@ -0,0 +1,146 @@
+                        System.out.println("Enter the name of the vegetable: ");
+                        String name = sc.next();
+                        System.out.println("Enter the price of the product: ");
+                        int price = sc.nextInt();
+                        System.out.println("Enter the weight of the product: ");
+                        int weight = sc.nextInt();
+                        System.out.println("Enter the quantity of the product: ");
+                        int quantity = sc.nextInt();
+                        System.out.println("Enter the validity date with format yyyy-MM-dd format");
+                        String validityDate = sc.next();
+                        System.out.println("Enter list of vitamins: ");
+                        String vitaminList = sc.next();
+                        productList.add(new Vegetable(id, name, price, weight, quantity, validityDate, vitaminList));
+                        System.out.println("Your product is in stock! ");
+                    } else {
+                        System.out.println("Invalid option! Please try again!");
+                    }
+                    break;
+                case 2:
+                    productList.forEach(System.out::println);
+                    System.out.println("=================================");
+                    System.out.println("Insert the date with format yyyy-MM-dd format ");
+                    String sellDate = sc.next();
+                    System.out.println("Insert the id of the product: ");
+                    int id = sc.nextInt();
+                        System.out.println("Insert the quantity: ");
+                        int soldQuantity = sc.nextInt();
+                        int quantity = product.getQuantity();
+                        int stockQuantity = stock.getQuantity();
+                        if(soldQuantity < quantity){
+                            orderList.add(new SoldItem(sellDate, soldQuantity));
+                            System.out.println("Your order has been created! ");
+                            continue;
+                        } else if(stockQuantity == 0) {
+                            orderList.add(new SoldItem(sellDate, product, soldQuantity));
+                            System.out.println("Your order has been created! ");
+                        }
+
+//                    Store store = new Store();
+//                    store.sellProduct(orderList);
+//                    store.updateStocQuantity();
+
+                    break;
+                case 3:
+                    System.out.println("Insert the date: ");
+                    String sellDate1 = sc.next();
+                    SoldItem sold = new SoldItem();
+                    sold.getSellDate();
+                    orderList.forEach(System.out::println);
+                    System.out.println("Your sales are: " + product.getId() + stock.getQuantity());
```

Comment:
you're printing all the orderList.. printing another message from a product and a stock... I don't think this step does what it's supposed to do